server/controllers/createTablesController.js

```diff
@@ -0,0 +1,34 @@
+import pool from './connect';
+import table from './dbTables';
+
+/*
+ * Class representing User Database Tables Handler
+ *
+ * @class TablesHandler
+ */
+
+class TablesHandler {
+  /*
+   * Reset or get a database table
+   *
+   * @static
+   * @param {object} req - The request object
+   * @param {object} res - The response object
+   * @returns {object} JSON object representing the response message
+   * @memberof TablesHandler
+   */
+  static createTable(req, res) {
+    pool.query(table.entriesTable)
+      .then(result => res.status(201)
+        .json({
+          result,
+          message: 'Created'
+        }))
+      .catch(err => res.status(500)
+        .json({
+          message: `from database ${err}`
+        }));
+  }
+}
+
```

Comment:
Instead of executing your queries in a controller, you can a script in `package.json` for this purpose.

Reply:
Thanks for spotting these errors out. I will implement the feedback right away.